CONTRIBUTING.md

```diff
@@ -2,5 +2,10 @@
 
 Chat with us in the [bugbug](https://chat.mozilla.org/#/room/#bugbug:mozilla.org) Matrix room.
 
-1. [Issues marked as `good-first-bug`](https://github.com/mozilla/bugbug/labels/good-first-bug) are self-contained enough that a contributor should be able to work on them.
+1. [Issues marked as good-first-bug](https://github.com/mozilla/bugbug/labels/good-first-bug)  are self-contained enough that a contributor should be able to work on them.
+
```

Comment:
are these changes okay ?

Reply:
The diff should show only the point number 3

Reply:
@jpangas gave some pointers above so according to that I made changes and added point number 3

Reply:
@srishtig2412 , I think you slightly misunderstood me. I meant you drop all changes we proposed. (by me and you) previously, then only include the part 3 suggested by @suhaibmujahid . :)

Reply:
Okay I have removed the first two points . I hope is it fine now

Reply:
https://github.com/mozilla/bugbug/blob/21a77f4e8d36868bbb62d9b81df3ddcba97e3b4c/CONTRIBUTING.md

Reply:
I have reset the changes ,please check

Reply:
There are linting errors, please cross check and see. 
<pre>
-3. If you have any problem, it could be already asnwerd in [Discussions](https://github.com/mozilla/bugbug/discussions), if not, feel free to start a new discussion in the [Q&A](https://github.com/mozilla/bugbug/discussions/categories/q-a) category.
\ No newline at end of file
+3. If you have any problem, it could be already asnwerd in [Discussions](https://github.com/mozilla/bugbug/discussions), if not, feel free to start a new discussion in the [Q&A](https://github.com/mozilla/bugbug/discussions/categories/q-a) category.
</pre>

Reply:
Run pre-commit:
```console
pre-commit run --all-files
```
Then it will work on the lint . Add the file(s) that was changed:
```console
git add .
```
Then finally commit those changes.
```console
git commit -m "Solve lint errors"
```
pre-commit should run once again and this time no errors should be shown. Confirm and see all checks have passed.
After they have passed, push to your remote branch.

Reply:
I am trying to do this but it's giving some kind of error .I'll complete this as soon as the error is recitifed

Reply:
Feel free to ask for help in discussions for help or in matrix

Reply:
Hello,  I  tried to resolve the errors ,please check once, if the changes are visible please let me know